paper.tex

```diff
@@ -476,137 +476,156 @@ \section*{Appendix}
 This schema, and its accompanying documentation,
 fully describe the entities in the model and their
 relationships and the required behaviour of implementations.
-Since the schema is definitive, we will not recapitulate the details
+Since the schema is definitive,
+\aprcomment{what exactly do we mean by ``definitive''?}
+we will not recapitulate the details
 here, but instead focus on the high level properties of the model and
 the rationale behind key design decisions.
 
-Below, we detail
-\begin{itemize}
-    \item The population genetics model in Section~\ref{sec:appendix-pop-gen-model}
-    \item The high-level model specification in Section~\ref{sec:appendix-spec}
-    \item The desired behaviour of Demes-format parsers in Section~\ref{sec:appendix-parsers}
-    \item And the intended scope of Demes in Section~\ref{sec:appendix-scope}
-\end{itemize}
+Below we detaail the population genetics model in 
+Appendix~\ref{sec:appendix-pop-gen-model},
+the high-level, human-readable model specification in
+Appendix~\ref{sec:appendix-spec}, demographic model parsers and their
+intended behaviour in Appendix~\ref{sec:appendix-parsers}, and the
+intended scope of Demes in Appendix~\ref{sec:appendix-scope}.
+The extended figures in Appendix~\ref{sec:appendix-figures} provides
+additional examples and illustrations of Demes usage.
 
-\section{Population genetics model details}\label{sec:appendix-pop-gen-model}
+\section{Population genetics model details}
+\label{sec:appendix-pop-gen-model}
 
 In Demes, demographic models consist of one or more interacting populations,
-or ``demes'', by which we mean a grouping of individuals that is convenient
-for modelling \citep{gilmour_demes_1939,gilmour_deme_1955}.
+or ``demes'', understood to be a collection of individuals that can be
+conveniently modelled using a defined set of rules and parameters
+\citep{gilmour_demes_1939,gilmour_deme_1955}.
 To avoid confusion with the name of the specification itself we will
 use the term ``population'' in this discussion, with the understanding that the
-terms are interchangeable for our purposes.
+terms are interchangeable.
 A population is defined as some collection of individuals that exists for
 some period of time, and has a well-defined size (i.e., number of individuals)
 during that time period. Individuals can move between populations
-either through ancestor-descendant relationships between the populations,
-or through processes of continuous or pulse migration.
-No other properties of the populations are specified in the model:
-we are concerned only with defining the populations, their sizes, and the
+either according to their ancestor-descendant relationships
+or through processes of continuous or pulse migrations.
```

Comment:
maybe simpler to say "processes involving migration"?

Reply:
Changed.